tools/packaging/common/istio-iptables.sh

```diff
@@ -424,12 +424,6 @@ if [ -n "${OUTBOUND_PORTS_EXCLUDE}" ]; then
   done
 fi
 
-if [ -z "${DISABLE_REDIRECTION_ON_LOCAL_LOOPBACK-}" ]; then
```

Comment:
you dont need to touch this file at all

Reply:
If i donot change this, envoy send traffic to podip will be intercepted by iptables and proxied to envoy again and again.